dashboard/src/components/users/UserPage.vue

```diff
@@ -64,11 +72,14 @@ limitations under the License.
         'numSkills',
         'userTotalPoints',
       ]),
+      showUserTags() {
```

Comment:
there is no need for this to be a computer property since it's just used for an if statement